sql_queries/PopulateSpatialSubset_TRAJECTORY.sql

```diff
@@ -0,0 +1,410 @@
+
+SET SEARCH_PATH = marvl3, public;
+	
+----AUV
+\echo 'AUV'
+INSERT INTO marvl3.spatial_subset(
+source_id,
+origin_id,
+"LONGITUDE",
+"LATITUDE",
+"TIME",
+"DEPTH",
+"TEMP",
+"PSAL",
+geom
+)
+SELECT 
+s.source_id,
+d.file_id,
+d."LONGITUDE",
+d."LATITUDE",
+d."TIME",
+d."DEPTH",
+d."TEMP",
+d."PSAL",
+d.geom
+FROM auv.auv_trajectory_st_data d,  "500m_isobath" p,source s
+WHERE ST_CONTAINS(p.geom, d.geom)
+AND s.table_name= 'auv_trajectory_st_data' 
+AND d."TIME" >= '1995-01-01' 
+AND d."TIME" < '2015-01-01'
+	
+---SOOP-CO2	
+\echo 'SOOP-CO2'
+INSERT INTO marvl3.spatial_subset(
+source_id,
+origin_id,
+"LONGITUDE",
+"LONGITUDE_QC",
+"LATITUDE",
+"LATITUDE_QC",
+"TIME",
+"TIME_QC",
+"DEPTH",
+"TEMP",
+"TEMP_QC",
+"PSAL",
+"PSAL_QC",
+geom
+)
+SELECT 
+s.source_id,
+d.cruise_id,
+d."LONGITUDE",
+d."LONGITUDE_quality_control",
+d."LATITUDE",
+d."LATITUDE_quality_control",
+d."TIME",
+d."TIME_quality_control",
+0,
+d."TEMP_1",
+d."TEMP_1_quality_control",
+d."PSAL",
+d."PSAL_quality_control",
+d.geom
+FROM soop_co2.soop_co2_trajectory_data d, "500m_isobath" p,source s
+WHERE ST_CONTAINS(p.geom, d.geom) 
+AND s.table_name='soop_co2_trajectory_data' 
+AND d."TIME" >= '1995-01-01' 
+AND d."TIME" < '2015-01-01'
+AND vessel_name IN('L''Astrolabe', 'Aurora Australis')	   
+
+DELETE "TEMP" FROM marvl3.spatial_subset WHERE source_id=19 -- delete Temperature data from SOOP-CO2 will be extracted from other schemas
+ 	
+
+
+---SOOP-TRV
+\echo 'SOOP-TRV'
+INSERT INTO marvl3.spatial_subset(
```

Comment:
I've already merged in master a sql query to process SOOP TMV and TRV so that they are binned and sub-sampled over time. Unless you are doing something I am missing, you may want to delete this.